src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String helloMates;
+    private int variable;
```

Comment:
```suggestion
    private int year;
```
left a comment in your Slack channel